### PR TITLE
Oreo Notification Fixed!

### PR DESCRIPTION
**Pull request by geekykant**
## Notification Fixes

For a better understanding of the notification issues on Oreo and above device, check this link. (https://medium.com/globallogic-latinoamerica-mobile/firebase-cloud-messaging-warning-updating-to-android-oreo-1343fe894bd5)

- New notification icon added
- Small code changes (tint to notification logo, meta adding in manifest.xml)

screenshot_2018-11-15-22-04-39-457_com mi android globallauncher

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -39,7 +39,7 @@
     <string name="skip_short_break">Skip Short Break</string>
     <string name="start_long_break">Start Long Break</string>
     <string name="skip_long_break">Skip Long Break</string>
-    <string name="skip_break">Skip Break</string>
```
Comment (piedcipher) [inline app/src/main/res/values/strings.xml:42]: This String has it's usages, I wonder why you removed it.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -39,7 +39,7 @@
...
     <string name="currently_running_service_type_key">CURRENTLY_RUNNING_SERVICE_TYPE</string>
-    <string name="pomodoro_completion_alert_message">Congrats, Pomodoro is over.\n\nIt\'s time for a break.</string>
```
Comment (piedcipher) [inline app/src/main/res/values/strings.xml:44]: This String has it's usages too, I wonder why you removed it.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (geekykant) [reply app/src/main/res/values/strings.xml:42]: Ohh sorry. It got removed when I rebased it.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (geekykant) [reply app/src/main/res/values/strings.xml:44]: Ohh sorry again. Same case!

**Comment by piedcipher** [comment]
And that article is for issues related to notification when FCM is used in project, but in this project FCM isn't used, so there aren't any issues related notifications in Oreo. I use an Oreo 8.1 device to deploy and test the app, and it works just fine. @geekykant

**Comment by numerative** [comment]
No need to create a new PR after fixes, @geekykant.

All you need to do is sync with upstream and push force your changes.

This will involve some interactive  rebasing and dropping of commits.

Please let me know if you need any help.

**Comment by geekykant** [comment]
Alright. let me try. Surely, I will ask if required. 👍

**Pull request closed without merging**